homeassistant/components/zha/__init__.py

```diff
@@ -110,9 +110,14 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     device_registry.async_get_or_create(
         config_entry_id=config_entry.entry_id,
         connections={
-            (dr.CONNECTION_ZIGBEE, str(zha_gateway.application_controller.ieee))
+            (
+                dr.CONNECTION_ZIGBEE,
+                str(zha_gateway.application_controller.state.node_info.ieee),
```

Comment:
as I look at this again... `zha_gateway.application_controller.state.node_info.ieee` is a lot. Should we make this a property on the gateway that just exposes this? something like `coordinator_ieee`? I'm good either way but this is in several spots.